cmd/age/age.1

```diff
@@ -0,0 +1,162 @@
+.Dd June 28, 2020
+.Dt AGE 1
+.Sh NAME
+.Nm age
+.Nd a simple, modern and secure file encryption tool
+.Sh SYNOPSIS
+.Nm
+.Fl r Ar recipient
+.Op Fl a
+.Op Fl o Ar output
+.Op Ar input
+.Nm
+.Fl -decrypt
+.Op Fl i Ar key
+.Op Fl o Ar output
+.Op Ar input
+.Sh DESCRIPTION
+In the first synopsis form,
+.Nm
+encrypts
+.Ar input
+to the specified
+.Ar recipient .
+In the second synopsis form,
+.Nm
+decrypts
+.Ar input .
+.Pp
+The following options are available:
+.Bl -tag -width Ds -compact
+.It Fl o , Fl -output Ar output
+Write the result to the file at path
+.Ar output .
+.It Fl a , Fl -armor
+Encrypt to a PEM encoded format.
+.It Fl p , Fl -passphrase
+Encrypt with a passphrase.
+.It Fl r , Fl -recipient Ar recipient
+Encrypt to the specified
+.Ar recipient .
+Can be repeated.
+.It Fl d , Fl -decrypt
+Decrypt the input to the output.
+.It Fl i , Fl -identity Ar key
+Use the private key file at path
+.Ar key .
+Can be repeated.
+.El
+.Pp
+The
+.Ar input
+argument is optional and defaults to standard input. Only a single file may be
+specified.
+If
+.Fl o
+is not specified then
+.Ar output
+defaults to standard output.
+.Pp
+The
+.Ar recipient
+can be an
+.Nm
+public key, as generated by 
+.Nm age-keygen ,
+.Po
+.Qq age1...
+.Pc
+or an SSH public key
+.Po
+.Qq ssh-ed25519 AAAA... ,
+.Qq ssh-rsa AAAA...
+.Pc .
+.Pp
+The identity
+.Ar key
+supplied to
+.Fl i
+/
+.Fl -identity
+is a path to a file with
+.Nm
+secret keys, one per line
+(ignoring "#" prefixed comments and empty lines), or to an SSH key file.
+Multiple keys can be provided, and any unused ones will be ignored.
+.Ss Multiple recipients
+Files can be encrypted to multiple recipients by repeating
+.Fl r
+/
+.Fl -recipient .
+Every recipient will be able to decrypt the file.
+.Ss Passphrases
+Files can be encrypted by using
+.Fl p
+/
+.Fl -passphrase .
+By default
+.Nm
+will automatically generate a secure passphrase. Passphrase protected files are
+automatically detected at decrypt time.
+.Ss SSH keys
+As a convenience feature,
+.Nm
+also supports encrypting to
+.Nm ssh-rsa
+and
+.Nm ssh-ed25519
+SSH public keys, and decypting with the respective private key file.
+.Po
+.Nm ssh-agent
+is not supported.
+.Pc
+.Pp
+Note that SSH key support employs more complex cryptography, and embeds a
+public key tag in the encrypted file, making it possible to track files that
+are encrypted to a specific public key.
+.Sh EXAMPLES
+Generate a secret key, encrypt data, and decrypt:
```

Comment:
I'm not sure if this was secret brilliance on your part or just a coincidence of copying over the README, but this conveniently also encourages people just following the examples blindly to frequently generate new keys, which seems like a net win to me.

Reply:
I shall attribute the brilliance here to Filippo. :-)